app/views/devise/two_factor_authentication_setup/index.html.slim

```diff
@@ -10,5 +10,6 @@ h1.heading = t('devise.two_factor_authentication.two_factor_setup')
   p#2fa-description = t('devise.two_factor_authentication.otp_setup')
   = f.input :phone, as: :tel, label: 'Phone Number', required: true,
       input_html: { 'aria-describedby' => '2fa-description' }
+  = f.input :phone_sms_enabled, as: :boolean, label: 'SMS Enabled?', input_html: { checked: true }
```

Comment:
We can actually do this now without waiting for the gem to be updated by making the `send_two_factor_authentication_code` a no-op method. We can then get rid of the UserOtpSender class and have the controller invoke the proper job based on the choice the user made and we can get rid of storing stuff in the session or DB.